cmd/unity-installer/cmd_list.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+type list struct{}
+
+func (l *list) Run(ctx commandContext) error {
+	latestReleases, err := ctx.releaseSource.FetchReleases(CLI.Platform, true)
+	if err != nil {
+		return err
+	}
+
+	versions := make([]string, len(latestReleases))
+	i := 0
+	for version := range latestReleases {
+		versions[i] = version
+		i++
+	}
```

Comment:
Not sure if I'm missing something, but would this work:

```suggestion
	versions := make([]string, 0, len(latestReleases))
	for version := range latestReleases {
		versions = append(versions, version)
	}
```
?